README.md
# How to install
- Navigate to `C:\<your server>\website\template\kaelthas`
- Extract the contents from the download into the directory of kaelthas.

# How to edit theme
- Navigate to `C:\<your server>\website\template\kaelthas\css`
- Open style.css & edit it however you please.

# How to edit templates
- Navigate to `C:\<your server>\website\template\kaelthas\tpl`
- Open any of the php files to edit them.

## [CHANGELOG.MD](https://github.com/etcroot/kaelthas-improved/blob/main/CHANGELOG.md)
Click the title to read the changelogs.

# Previews
![Home Page](https://i.imgur.com/rxx1nDM.png)
![Registration Page](https://i.imgur.com/jxq3tuA.png)

# Credits
Original Author: [masterking32](https://masterking32.com)


CHANGELOG.md
### 2022/09/03 - v1.0.5
- Formatted documents for easier readability.

### 2022/09/03 - v1.0.4
- Added a post entry with the improvement information.

### 2022/09/03 - v1.0.3
- Website font can now be changed through a variable, if you wish to change the font linked it's top of the style.css file.
You can find more fonts at: [Google Fonts](https://fonts.google.com/)

### 2022/09/03 - v1.0.2
- Another typo has been fixed in main.php.

### 2022/09/03 - v1.0.1
- Fixed a typo from the original author on the Currently Online Players page.

### 2022/09/03 - v1.0.0
- Converted colours to CSS variables for easier editing of the color scheme.
- Added a variable for background image for easy editing.
- Completely overhauled the color scheme.
- Added flex boxes instead of normal CSS layouts, this is for more responsive & modern design (Will be expanded upon).


tpl/posts.php
<?php

/**
 * Created by Amin.MasterkinG
 * Website : MasterkinG32.CoM
 * Email : lichwow_masterking@yahoo.com
 * Date: 11/26/2018 - 8:36 PM
 */
?>
<div class="box1" style="margin-top: 10px;">
    <h3 class="box-title">Template improvement by etcroot</h3>
    <hr class="divider">
    <div class="box-content">
        <p>You can read more about the template improvement at the <a href="https://github.com/etcroot/kaelthas-improved" target="_blank">Github Page</a>
        <p>Here's the changelog: <a href="https://github.com/etcroot/kaelthas-improved/blob/main/CHANGELOG.md" target="_blank">Changelogs</a></p>
    </div>
</div>
<div class="box1" style="margin-top: 10px;">
    <h3 class="box-title">Welcome to Single Player Project - LegionV2 server!</h3>
    <hr class="divider">
    <div class="box-content">
        <p>
            You can create your game account under the Registration button on the top.
            Change the server address in World of Warcraft\WTF\Config.wtf file and set to 127.0.0.1.
        </p>
        <code>SET portal "127.0.0.1"</code>
        </p>
        <p>
            You need to configure your game version in bnetserver.conf and worldserver.conf files before start it.
            You can find your game build number bottom left on the ingame login screen.
        </p>
    </div>
</div>

tpl/howtoconnect.php
<?php

/**
 * Created by Amin.MasterkinG
 * Website : MasterkinG32.CoM
 * Email : lichwow_masterking@yahoo.com
 * Date: 11/26/2018 - 8:36 PM
 */
?>
<div class="box1">
    <h3 class="box-title">Welcome to Single Player Project - LegionV2 server!</h3>
    <hr class="divider">
    <p>1. First of all, you must create an account. The account is used to log into both the game and our website. Click
        here to open the registration page.</p>
    <p>2. Install World of Warcraft. You can download it (legally) from here: Windows or Mac. Make sure to upgrade to
        our current supported patch, which is 7.3.5 (build 26124-26972). Patch mirrors can be found here. </p>
    <p>3. Open up the "World of Warcraft" directory. The default directory is "C:\Program Files\World of Warcraft". When
        you've found it, open up the directory called "data", then go into the directory called either enUS or enGB,
        depending on your client language.</p>
    <p>5. Erase all text and change it to:<br />
        <code>set portal "<?php echo get_config('realmlist'); ?>"</code>
    </p>
    <p>You may now start playing! If you need any help, do not hesitate to create a support ticket.</p>
</div>

tpl/header.php
<?php

/**
 * Created by Amin.MasterkinG
 * Website : MasterkinG32.CoM
 * Email : lichwow_masterking@yahoo.com
 * Date: 11/26/2018 - 8:36 PM
 */
?>

<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="generator" content="MasterkinG32.CoM" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta property="description" content="<?php echo $antiXss->xss_clean(get_config("page_title")); ?>">
    <meta name="description" content="<?php echo $antiXss->xss_clean(get_config("page_title")); ?>">
    <title><?php echo $antiXss->xss_clean(get_config("page_title")); ?></title>
    <link rel="stylesheet" href="<?php echo $antiXss->xss_clean(get_config("baseurl")); ?>/template/<?php echo $antiXss->xss_clean(get_config("template")); ?>/css/bootstrap.min.css">
    <link rel="stylesheet" href="<?php echo $antiXss->xss_clean(get_config("baseurl")); ?>/template/<?php echo $antiXss->xss_clean(get_config("template")); ?>/css/bootsnav.css">
    <link rel="stylesheet" href="<?php echo $antiXss->xss_clean(get_config("baseurl")); ?>/template/<?php echo $antiXss->xss_clean(get_config("template")); ?>/css/animate.css">
    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
    <link rel="stylesheet" href="<?php echo $antiXss->xss_clean(get_config("baseurl")); ?>/template/<?php echo $antiXss->xss_clean(get_config("template")); ?>/css/style.css">
    <script src="<?php echo $antiXss->xss_clean(get_config("baseurl")); ?>/template/<?php echo $antiXss->xss_clean(get_config("template")); ?>/js/jquery-3.3.1.min.js"></script>
    <script src="<?php echo $antiXss->xss_clean(get_config("baseurl")); ?>/template/<?php echo $antiXss->xss_clean(get_config("template")); ?>/js/bootstrap.min.js"></script>
    <script src="<?php echo $antiXss->xss_clean(get_config("baseurl")); ?>/template/<?php echo $antiXss->xss_clean(get_config("template")); ?>/js/bootsnav.js"></script>
    <script src="<?php echo $antiXss->xss_clean(get_config("baseurl")); ?>/template/<?php echo $antiXss->xss_clean(get_config("template")); ?>/js/popper.min.js"></script>
</head>

<body class="wallpaper">
    <div class="content1">
        <div class="container">
            <nav class="navbar navbar-default brand-center bootsnav">
                <div class="container">
                    <!-- Start Header Navigation -->
                    <div class="navbar-header">
                        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-menu">
                            <i class="fa fa-bars"></i>
                        </button>
                        <a class="navbar-brand" href="/"><img src="<?php echo $antiXss->xss_clean(get_config("baseurl")); ?>/template/<?php echo $antiXss->xss_clean(get_config("template")); ?>/images/wow-logo.png" class="logo" alt=""></a>
                    </div>
                    <!-- End Header Navigation -->
                    <!-- Collect the nav links, forms, and other content for toggling -->
                    <div class="collapse navbar-collapse" id="navbar-menu">
                        <ul class="nav navbar-nav" data-in="fadeInDown" data-out="fadeOutUp">
                            <li><a href="./index.php">Home</a></li>
                            <li><a onclick="$('#register').trigger('click')">Register</a></li>
                            <li><a onclick="$('#howtoconnect').trigger('click')">How to connect</a></li>
                            <li><a onclick="$('#serverstatus').trigger('click')">Server Status</a></li>
                            <li><a onclick="$('#topplayers').trigger('click')">Top Players</a></li>
                            <li><a onclick="$('#contact').trigger('click')">Contact us</a></li>
                        </ul>
                    </div><!-- /.navbar-collapse -->
                </div>
            </nav>